.pylintrc

```diff
@@ -0,0 +1,386 @@
+[MASTER]
+
+# Specify a configuration file.
+#rcfile=
+
+# Python code to execute, usually for sys.path manipulation such as
+# pygtk.require().
+#init-hook=
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=.git,dispersy,pymdht,libnacl,data
```

Comment:
I think something went wrong with github. In any case: ignoring dispersy on dispersy will not leave much I think :D